edx_api/grades/__init__.py

```diff
@@ -0,0 +1,76 @@
+"""
+edX Grades REST API client class
+"""
+from requests.exceptions import HTTPError
+from six.moves.urllib.parse import urljoin  # pylint: disable=import-error
+
+from edx_api.enrollments import CourseEnrollments
+from .models import CurrentGrade, CurrentGrades
+
+
+class UserCurrentGrades(object):
+    """
+    edX student certificates client
+    """
+
+    def __init__(self, requester, base_url):
+        """
+        Args:
+            requester (Requester): an authenticated objects for requests to edX
+            base_url (str): string representing the base URL of an edX LMS instance
+        """
+        self.requester = requester
+        self.base_url = base_url
+
+    def get_student_current_grade(self, username, course_id):
```

Comment:
we do not do validation about the structure of the data in this library: we rely on the fact that edX will handle errors properly with proper HTTP statuses